cluster/images/hyperkube/Makefile

```diff
@@ -21,6 +21,7 @@ REGISTRY?=gcr.io/google-containers
 ARCH?=amd64
 HYPERKUBE_BIN?=_output/dockerized/bin/linux/$(ARCH)/hyperkube
 
+# TODO(dixudx): update the tag to 0.2 when the image get pushed
```

Comment:
@dixudx That's okay, just do it now.
The CI will go green when we retest and @ixdy has pushed the images based on this PR